packages/lib/actions/clearLocalStorage/action.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "../../schemas/libAction.schema.json",
-  "name": "Clear Local Storage",
+  "name": "Clear local storage",
```

Comment:
I'm quite sure this will break all references to this action. I believe we introduced a display name or label in formulas. Perhaps we can do something similar for actions?

Reply:
I believe it only broke when we changed the actual file names before?

If this is the case though, I'd rather just fix it with CSS, but it looks like there may be a bug in the CSS parser as ::first-letter wasn't working when I tried it earlier today.